internal/security/secretstore/certs.go

```diff
@@ -178,26 +174,7 @@ func (cs *Certs) ReadFrom(certPath string, keyPath string) (*CertPair, error) {
 	return &cp, nil
 }
 
-func GetAccessToken(filename string) (string, error) {
-	a := auth{}
```

Comment:
Since struct `auth` is no longer used, it can be removed.

Reply:
Fixed.